packages/RdfService/README.md

```diff
@@ -2,7 +2,7 @@
 
 RdfService is a helper library that abstracts away the complexity of interacting with RDF triplestores, providing basic CRUD abilities.
 
-For futher information on how to use the library see the [API documentation](https://needtocreate.com)
+For futher information on how to use the library see the [API documentation](README.md#API)
 
 ## Install
```

Comment:
As per the main README.md and the OntologyService README.md, in the Usage section, below, the code example just needs a one-liner to introduce it, e.g. "A simple example that runs a query on the 'ontology' rdf service:".